operations/helm/charts/mimir-distributed/templates/store-gateway/store-gateway-statefulset.yaml

```diff
@@ -57,8 +57,20 @@ spec:
     metadata:
       labels:
         {{- include "mimir.podLabels" $args | nindent 8 }}
+        {{- if (eq $rolloutZone.noDownscale true )}}
+        grafana.com/no-downscale: {{$rolloutZone.noDownscale}}
+        {{- else }}
+        grafana.com/prepare-downscale: {{$rolloutZone.prepareDownscale}}
+        {{- end }}
       annotations:
         {{- include "mimir.podAnnotations" $args | nindent 8 }}
+         {{- if $rolloutZone.prepareDownscale }}
+        grafana.com/prepare-downscale-http-path: store-gateway/prepare-shutdown
+        grafana.com/prepare-downscale-http-port: {{ include "mimir.serverHttpListenPort" . }}
+        {{- end-}}
```

Comment:
i think this tag is invalid - it should be `{{- end -}}` (extra space) It's also invalid for the other two templates